src/main/java/bio/terra/flight/exception/InvalidIngestStrategyException.java

```diff
@@ -0,0 +1,9 @@
+package bio.terra.flight.exception;
+
+import bio.terra.exception.BadRequestException;
+
+public class InvalidIngestStrategyException extends BadRequestException {
+    public InvalidIngestStrategyException(String message) {
+        super(message);
+    }
```

Comment:
Wasn't sure how many of the constructors should be added here. This is the only one we need to the existing code.

Reply:
That is fine. We never supply all of the overloads; way too much useless gibberish :)